Hotfix/sidenav search banner push

acacase opened the pull request:
- Added .bu-banner to .nav-open and .search-open styles
- Increased the search-height variable

Example URL:
http://responsi.cms-devl.bu.edu/responsi-sidenav/

ashleykolodziej commented on css-dev/burf-theme/layout/_navigation.scss:
```diff
@@ -868,7 +869,9 @@ $_search-height: 103px;
+		.content
+		 {
 			@include breakpoint( $left-nav-desktop ) {
 				left: $width-side-nav - 60px;
```
Is the 60px here referring to the width of the search button? If so, it'd be great to use `$nav-toggle-size` here! https://github.com/bu-ist/responsive-foundation/blob/develop/css-dev/burf-theme/layout/_navigation.scss#L384

acacase replied:
Not sure what the 60px is referring to, it was there to begin with for the .content selector. I just added the .bu-banner selector to it.

I don't think it refers to the search button width because this chunk of code pushes the banner and content to the right when the sidenav is toggled open.

ashleykolodziej replied:
Ah! Got it. Then it's probably best to leave it alone. That makes sense.

ashleykolodziej commented:
OH - one more thing - can you add a bullet point to the changelog that you fixed a bug with the left hand layout and BU Banners? You can put it under an "Unreleased" heading (most of what's in there is probably under tags like 2.0.0, etc)

acacase commented:
@ashleykolodziej I see the "Unreleased" changes in the Framework change log. Should I also add something to the Foundation change log?

ashleykolodziej commented:
Yes please!

acacase commented:
@ashleykolodziej Would I start a Unreleased heading in the Foundation change log or would I create a list with a version number. So for example would I write:

## Unreleased
* fixed a bug with the left hand nav layout and BU Banners

or

## 2.0.2
* fixed a bug with the left hand nav layout and BU Banners

ashleykolodziej commented:
Start with unreleased - we'll change it to the release number when we do the release!
